packages/webapp/src/utils/constants.js

```diff
@@ -234,7 +237,9 @@ const redisKeys = {
   REGISTRATION_TASK_DETAILS,
   LAND_OWNERSHIP_KEY,
   METRIC_UPLOADED_ANSWER,
-  REFERER
+  REFERER,
+  LAND_OWNER_EMAIL,
```

Comment:
Rediskeys normally have a distinct key rather than using a route. for LAND_OWNER_EMAIL and CONFIRM_OWNER_EMAIL